app/src/main/res/values/help.xml

```diff
@@ -55,10 +48,9 @@ To exit Search Mode, tap the small cross in the corner of the screen.&lt;br/&gt;
 Time Travel mode enables you to use Sky Map to see the sky at times
```

Comment:
Improved wording: "Time Travel Mode enables you to see the sky at times..." (don't need to reference the app in the sentence)

Reply:
.